model/event/subscriptions.go

```diff
@@ -33,6 +34,13 @@ var (
 	subscriberWithRegexRegexSelectorsKey = bsonutil.MustHaveTag(subscriberWithRegex{}, "RegexSelectors")
 )
 
+type OwnerType string
+
+const (
+	OwnerTypePerson  OwnerType = "person"
+	OwnerTypeProject           = "project"
```

Comment:
In this case, I believe that OwnerTypeProject is being declared as a string, and you need add the type annotation.

Reply:
Proof: https://play.golang.org/p/TKFXtme5cge

Reply:
added